.github/workflows/Dockerfile.ci.alpine

```diff
@@ -39,17 +39,24 @@ RUN if [ "${TARGETPLATFORM}" = "linux/arm/v7" ]; then \
 # Web and App dir
 COPY docker-entrypoint.sh /docker-entrypoint.sh
 COPY lldap_config.docker_template.toml /lldap/
+COPY web/index_local.html web/index.html
 RUN cp target/lldap /lldap/ && \
     cp target/migration-tool /lldap/ && \
     cp -R web/index.html \
           web/pkg \
           web/static \
           /lldap/app/
 
+WORKDIR /lldap
 RUN set -x \
     && for file in $(cat /lldap/app/static/libraries.txt); do wget -P app/static "$file"; done \
     && for file in $(cat /lldap/app/static/fonts/fonts.txt); do wget -P app/static/fonts "$file"; done \
-    && chmod a+r -R .
+    && chmod a+r -R . \
+    && ls -al . \
+    && ls -al app/ \
+    && cat app/index.html \
```

Comment:
Instead of `cat` maybe `[ -f app/index.html ]`? We just want to test whether the file exists.
Similarly, the `ls` can be replaced with existence tests like `[ -d app/ ]`. And we don't need to test whether `.` exists.

Reply:
Do we need test every file? Because practically we need to see is everything downloaded correctly, but also we can remove it. Or we can just remove it, as after few runs it downloaded and copied fine.

Reply:
yeah, I think you can do without testing the files.